core/src/main/scala/kafka/server/SharedServer.scala

```diff
@@ -97,6 +101,10 @@ class SharedServer(
   @volatile var raftManager: KafkaRaftManager[ApiMessageAndVersion] = _
   @volatile var brokerMetrics: BrokerServerMetrics = _
   @volatile var controllerMetrics: QuorumControllerMetrics = _
+  @volatile var loader: MetadataLoader = _
+  val snapshotsDiabledReason = new AtomicReference[String](null)
```

Comment:
Is this typo upstream too?

Reply:
Yup -- https://github.com/apache/kafka/blob/9089a4ddb9cbde195862514f027ced594883b2d9/core/src/main/scala/kafka/server/SharedServer.scala#L105

I'll leave it for now and we can fix upstream (its also in trunk)